emily/lambda/src/errors.rs

```diff
@@ -0,0 +1,166 @@
+use std::fmt;
+use aws_lambda_events::apigw::ApiGatewayProxyResponse;
+use aws_lambda_events::encodings::Body;
+use emily::models;
+use crate::common;
+
+#[derive(Debug)]
+pub enum EmilyApiError {
+    BadRequest(String),
+    // Forbidden(String), // Currently unused.
+    // NotFound(String), // Currently unused.
+    // Conflict(String), // Currently unused.
+    // NotImplemented(String), // Currently unused.
+    // Throttling(String), // Handled by the gateway, here for completeness.
+    Service(Result<String, Box<dyn std::error::Error>>),
```

Comment:
I think we should a type other than `Result` here (or break up this variant into two).